src/replit/ai/modelfarm/txt2image_model.py

```diff
@@ -0,0 +1,149 @@
+import os
+import requests
+import aiohttp
+from .model import Model
+from .structs import Txt2ImgModelResponse, LoraWeight, ImageSizeInput
+from typing import Optional, Any, Dict, Literal
+
+DEFAULT_LORA_URL = "https://110602490-lora.gateway.alpha.fal.ai"
```

Comment:
Shouldn't this be part of https://github.com/fal-ai/replit-ai-python/blob/8af6a3c579367b21e541a9fe7d00040856130de7/src/replit/ai/modelfarm/config.py#L4-L9?